From 285bc9594cf1f615fb73705a68d67c6824272ac4 Mon Sep 17 00:00:00 2001
From: Vasiliy Stelmachenok <ventureo@yandex.ru>
Date: Tue, 31 Oct 2023 11:04:44 +0300
Subject: [PATCH 021/243] Kconfig.preempt: Remove CONFIG_EXPERT dependency from
 PREEMPT_RT

It's just a headache to have a bunch of garbage in your config for the
sake of enabling one option. It makes it much easier to keep our config
and maintain it relative to other cachy kernels.

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 kernel/Kconfig.preempt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/Kconfig.preempt b/kernel/Kconfig.preempt
index 88c594c6d7fc..2471a502c52a 100644
--- a/kernel/Kconfig.preempt
+++ b/kernel/Kconfig.preempt
@@ -91,7 +91,7 @@ endchoice
 
 config PREEMPT_RT
 	bool "Fully Preemptible Kernel (Real-Time)"
-	depends on EXPERT && ARCH_SUPPORTS_RT && !COMPILE_TEST
+	depends on ARCH_SUPPORTS_RT && !COMPILE_TEST
 	select PREEMPTION
 	help
 	  This option turns the kernel into a real-time kernel by replacing
-- 
2.55.0

